backend/dal/Configuration/MotiClassificationConfiguration.cs

```diff
@@ -0,0 +1,31 @@
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using Pims.Dal.Entities;
+
+namespace Pims.Dal.Configuration
+{
+    /// <summary>
+    /// MotiClassificationConfiguration class, provides a way to configure property classifications in the database.
+    ///</summary>
+    public class MotiClassificationConfiguration : LookupEntityConfiguration<MotiClassification, int>
+    {
+        #region Methods
+        public override void Configure(EntityTypeBuilder<MotiClassification> builder)
+        {
+            builder.ToTable("MotiClassifications");
```

Comment:
I think these should probably be just `Classifications`.  Not sure it makes sense to have different types of classifications now.